demo/src/components/Cart/Cart.js

```diff
@@ -20,6 +20,7 @@ export const computePrice = (items) => {
   return totalPrice;
 };
 
+
```

Comment:
Nit: Unnecessary line break.  Only a single break is required between any two statements:
https://google.github.io/styleguide/jsguide.html#jsdoc-whitespace

Reply:
Removed